tests/test_api_analyses.py

```diff
@@ -64,56 +65,76 @@ def test_kpidemo_sources(session: MVG, waveform_source_with_measurements):
 
 
 def test_kpidemo_analysis(session, waveform_source_with_measurements):
-    kpi = session.request_analysis(waveform_source_with_measurements, "KPIDemo")
-    session.wait_for_analyses([kpi["request_id"]])
-    status = session.get_analysis_status(kpi["request_id"])
+    request = session.request_analysis(waveform_source_with_measurements, "KPIDemo")
+    request_id = request["request_id"]
+
+    session.wait_for_analyses([request_id])
+    status = session.get_analysis_status(request_id)
     assert status == "successful"
-    results = session.get_analysis_results(kpi["request_id"])
+    results = session.get_analysis_results(request_id)
     assert len(results.keys()) == 7
     assert results["status"] == "successful"
     assert results["feature"] == "KPIDemo"
     kpi_results = results["results"]
     assert len(kpi_results.keys()) == 2
     assert len(kpi_results["acc"].keys()) == 7
 
+    # Delete analysis
+    session.delete_analysis(request_id)
+
 
 @pytest.mark.skip(reason="callback feature")
 def test_callback(session, callback_server, waveform_source_with_measurements):
-    req = session.request_analysis(
+    request = session.request_analysis(
         waveform_source_with_measurements, "KPIDemo", callback_url=callback_server
     )
-    req_id = req["request_id"]
-    session.wait_for_analyses([req_id])
-    status = session.get_analysis_status(req_id)
+    request_id = request["request_id"]
+    session.wait_for_analyses([request])
+    status = session.get_analysis_status(request)
```

Comment:
```suggestion
    session.wait_for_analyses([request_id])
    status = session.get_analysis_status(request_id)
```

This test is skipped therefore this did not lead to an error. Do you know why is it skipped?

Reply:
Very good catch. Sorry, my bad. Missed this one while bulk replacing in VSCode. This belonged to the test `test_callback` which itself was skipped.